data-processing/common/commands/fetch_s2_data.py

```diff
@@ -59,13 +59,15 @@ def _mk_api_request(self, arxivId: ArxivId) -> requests.Response:
         headers = None
 
         if self._partner_api_token:
-            base_url = "partner.semanticscholar.org"
             headers = {"x-api-key": self._partner_api_token}
 
+        references_fields = ["authors", "title", "externalIds", "venue", "year"]
+        fields_query = "fields=references." + ",references.".join(references_fields)
```

Comment:
This took me a second to understand, mainly because of the leading `fields=references+` bit. What do you think about this?

```python
fields = ["references." + f for f in [ "authors", "title", "externalIds", "venue", "year" ] ]
```

Also I'm omitting the query string argument name. More on that below.